SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSchemaGenerator.h

```diff
@@ -6,7 +6,13 @@
 
 DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKSchemaGenerator, Log, All);
```

Comment:
Not related to this PR, but maybe we should consider wrapping these functions in some namespace?

Reply:
Yeah, we could put these in `SpatialGDK` namespace.